locomotion-sdk/src/Components/PageHeader/index.js

```diff
@@ -27,7 +27,6 @@ const PageHeader = ({
         </SkipButtonText>
       </SkipButton>
       )}
-    {children}
```

Comment:
I think I need this

Reply:
i changed it, i dont really think that needs to be part of the mani header anyway, but i fixed it so it looks the same still.